.github/workflows/sauce-ie11.yaml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   tests-sauce-ie11:
     # Don't run on forks. Sauce tests need access to secrets.
-    if: github.repository == 'lit/lit'
```

Comment:
Can you explain this one? This is true on forks? IMO, worth a comment on the new version why it isn't the seemingly obvious `.repository == 'lit/lit'`

Reply:
We want to specifically compare the PR's head repo (which could be a fork) and the base repo (lit/lit). PRs whose head repo is a fork would fail this condition and skip.

The existing `github.repository == 'lit/lit'` will always be true for PRs open in this repository.

Reply:
Ah... Thanks for the explanation!